apps/ffdweb-site/src/app/_components/PageHeader.tsx

```diff
@@ -0,0 +1,100 @@
+import Image from 'next/image'
+import type { StaticImageData } from 'next/image'
+
+import { Button } from '@filecoin-foundation/ui/Button'
+import { Heading } from '@filecoin-foundation/ui/Heading'
+import { buildImageSizeProp } from '@filecoin-foundation/utils/buildImageSizeProp'
+import type { CTAProps } from '@filecoin-foundation/utils/types/ctaType'
+import { clsx } from 'clsx'
+
+import { BASE_DOMAIN } from '@/constants/siteMetadata'
+
+type TitleProps = {
+  children: string
+  fontVariant?: PageHeaderFontVariant
+}
+
+export type StaticImageProps = {
+  data: StaticImageData
+  alt: string
+}
+
+type PageHeaderImageProps = StaticImageProps
+
+type PageHeaderFontVariant = 'default' | 'homepage'
+
+type PageHeaderProps = {
+  title: TitleProps['children']
+  image: PageHeaderImageProps
+  cta?: CTAProps
+  mobileLayout?: 'content-above-image' | 'image-above-content'
```

Comment:
Why do we need this prop? Is it because of this? [>>](https://www.figma.com/design/P67wEFE8S7ay0b7FErfqPE?node-id=5175-4177#1137671214)

Reply:
Ah yes, I was talking to @filipagr and she wanted to have homepage a bit different than the others. But we can use the `isHomepage` from the set the mobile order? And we remove `mobileLayout` prop?